testing/tools/integration/integration.py

```diff
@@ -1066,6 +1090,8 @@ def pipeline_test(generator, expected, command, workers=1, sources=1,
     then expected should be [1,1,1,2,2,2].
     """
 
+    if resilience_dir == None:
```

Comment:
nit: this should be `if resilience_dir is None:` for idiomatic Python (e.g. identity instead of equality test, since None is a singleton)